Automattic/jetpack
# Adds an IDC simulation module to the Debug Helper.

This change adds a new module to the Debug Helper module: 
![Screenshot from 2021-09-24 22-52-49](https://user-images.githubusercontent.com/374293/134732460-89ba108e-37d4-427d-b8e3-7ce07959524c.png)

So far the module is pretty simple - it lets you avoid adding filters, while simply enabling a simulated identity crisis using the `jetpack_sync_site_url` filter.

#### Changes proposed in this Pull Request:
* Added an IDC simulation module to the Debug Helper.

#### Jetpack product discussion
* N/A

#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
* Build the `jetpack-debug` or enable it using Jetpack Beta.
* Enable the IDC simulator module in the menu.
* Enable the IDC simulation in the UI.
* Create a draft post,
* Observe your site in IDC safe mode.
